src/plugins/dashboard/public/dashboard_actions/export_csv_action.tsx

```diff
@@ -41,7 +41,7 @@ const isApiCompatible = (api: unknown | null): api is ExportCsvActionApi =>
 export class ExportCSVAction implements Action<ExportContext> {
   public readonly id = ACTION_EXPORT_CSV;
   public readonly type = ACTION_EXPORT_CSV;
-  public readonly order = 18; // right after Export in discover which is 19
+  public readonly order = 17; // right after Export in discover which is 19
```

Comment:
Actually this change in order isn't necessary anymore since I made `Explore in Discover` a hover action, so they're sorted in the same menu anymore. I'll restore it to 18, but I'll remove the comment since `Explore in Discover` doesn't show in the more menu next to `Download CSV` anymore.